service_management/test/controllers/analysis/SampleObjectControllerIntegrationSpec.scala

```diff
@@ -188,6 +189,39 @@ class SampleObjectControllerIntegrationSpec
       validateSampleObject(ujs, res.json)
     }
 
+    "delete a specific sample object" in {
+      val all = getAllForTestMuseum
+
+      val expJs    = (all.json \ 2).as[JsObject]
+      val objectId = (expJs \ "objectId").as[String]
+
+      val res =
+        wsUrl(deleteUrl(mid)(objectId)).withHeaders(token.asHeader).get().futureValue
+
+      res.status mustBe OK
+    }
+
+    "try to delete a objectid that not exists, return NotFound" in {
```

Comment:
this test description doesn't read very well when put into the surrounding context
```
"Invoking the sample object controller API" should "try to delete a objectid that not exists, return NotFound"
```

It sort of inverts the intention... a better phrasing could for example be

```
"Invoking the sample object controller API" should "return 404 when trying to delete a non-existing sample object"
```